eventstore/store.go

```diff
@@ -205,6 +233,16 @@ func (s *Store) RegisterSchema(name string, schema string) (*Model, error) {
 	}
```

Comment:
Now I wonder if erroring on existing schema is the correct thing.
I guess we're not sure about how the client will know if do it or not... I guess by his own means of something similar. It can leverage `store.ThreadId()` or something that gives a clue (or maybe we can create a cheap func in the future).

Anyway, not mentioning to make something now... but just a flashing thought.

Reply:
I was thinking the same.... hard to say!